conf/variant/common/local.conf

```diff
@@ -20,3 +20,7 @@ DIRFILES_pn-nativesdk-cmake = "1"
 
 # Don't install any extra devel packages just because we're running on QEMU
 POKYQEMUDEPS_remove = "packagegroup-core-device-devel"
+
+# Don't build .hddimg and .iso image formats as we don't need it.
```

Comment:
No need for the `.` at the end of the line here. And s/it/them

Reply:
We are using `.rpi-sdimg` and `.wic` for pi and NUC, respectively.